Fix exception raised when load an certificate with UTF-8 characters (… (#1007)
pyca/pyopenssl: CHANGELOG.rst

Changes:
^^^^^^^^

> - Fixed exception raised when load an certificate with UTF-8 characters on ``Openssl.crypto.load_certificate``

Reviewer: ```suggestion
- Fixed exception raised when loading a certificate with UTF-8 characters in ``Openssl.crypto.load_certificate``
```
